crypto/evp/ctrl_params_translate.c

```diff
@@ -1384,16 +1384,16 @@ static int fix_rsa_pss_saltlen(enum state state,
             if (strcmp(ctx->p2, str_value_map[i].ptr) == 0)
                 break;
         }
-        if (i == OSSL_NELEM(str_value_map)) {
-            ctx->p1 = atoi(ctx->p2);
-        } else if (state == POST_CTRL_TO_PARAMS) {
+        int val = (i == OSSL_NELEM(str_value_map)) ? atoi(ctx->p2) :
```

Comment:
The declaration must be at the beginning of the block. This is the cause of the CI failures.